package.json

```diff
@@ -726,8 +726,10 @@
         "vscode:prepublish": "webpack --mode production",
         "webpack": "webpack --mode development",
         "webpack-dev": "webpack --mode development --watch",
-        "test-compile": "tsc -p ./",
```

Comment:
Is there a reason this is removed? I don't even know what it is for, but still would like to understand why we removed it :)

Reply:
guessing from its name, it is supposed to test whether our TS code compiles or not, but it's exactly same as the another existing `compile` command.
Newly added `compile-dry-run` effectively supersedes that.